### ISSUE TITLE
BUG: Image Encoding Parameter is not read as expected

### ISSUE THREAD

**Issue by tomcreutz** [basler/pylon-ros-camera#219]
### Issue description

When launching the Pylon ROS2 Camera Node Component, the image_encoding parameter is not read. Thus it will be automatically set to e.g. mono8.

I created a pull request with a suggested change: https://github.com/basler/pylon-ros-camera/pull/218

Note that this bug might also apply to other parameters that are not declared (e.g. binning_x and binning_y), but I haven't fixed them in this pull request as we don't use them in our setup right now.

### Code example about how to reproduce the issue

Basically set the image_encoding parameter in the parameter list of your launch file to any value (e.g. "bayer_grbg8"), then launch the component Node.

### Error message

```shell
[basler.pylon.ros2.pylon_ros2_camera_base]: No image encoding provided -> Will use 'mono8' or 'rgb8' as fallback
```

### Hardware setup description

- 

### Runtime information

```shell
ROS2 Humble
```

### Is your camera operational with the Basler pylon Viewer on your platform?

Yes

**Comment by FrancoisPicardDTI** [comment]
Hello @tomcreutz 
The issue is now fixed and I made a pass on all the other parameters.
Thank you very much!
